packages/gafl-webapp-service/src/pages/renewals/identify/identify.njk

```diff
@@ -22,9 +22,11 @@
             'string.pattern.base': { ref: '#postcode', text: 'Your postcode doesn\'t look right. Check and enter again' }
         },
         'referenceNumber' : {
+            'string.empty': { text: 'Enter the last six digits of your licence number', ref: '#ref' },
+            'string.pattern.base': { text: 'The last six digits of your licence number doesn\'t look right. Check and enter again', ref: '#ref' },
             'string.invalid': {
-                text: "The details you entered do not match those we have on record for licence number " + data.referenceNumber + " Please check you have entered the correct date of birth and postcode",
-                href: "#date-of-birth"
+                text: 'The details you entered do not match those we have on record for licence number ' + payload.referenceNumber + '. Please check you have entered the correct date of birth and postcode',
```

Comment:
For security reasons we should not be differentiating between an invalid postcode/dob for a known licence number or where the licence number does not exist.

Reply:
We don't - the postcode and dob are validated as simple business rules prior to authentication. Any validation errors specific to either postcode or DoB are purely atomic and do not reference the permission.